.circleci/config.yml

```diff
@@ -29,6 +29,33 @@ jobs:
             if [ "${CIRCLE_BRANCH}" == "master" ]; then
               echo "$DOCKERHUB_PASS" | docker login -u "$DOCKERHUB_USERNAME" --password-stdin
               docker push valhalla/valhalla:build-latest
+            elif [[ ! -z "${CIRCLE_TAG}" ]]; then
```

Comment:
circle changes are a work in progress. probably need to spend a few rounds on this yet